databricks_asset_bundles/02.train_model.py

```
@@ -0,0 +1,188 @@
+        model=pipeline,
+        flavor=mlflow.sklearn,
+        artifact_path="lightgbm-pipeline-model-fe",
+        training_set=training_set,
+        signature=signature,
+    )
+
+# COMMAND ----------
+
+
+model_uri=f'runs:/{run_id}/lightgbm-pipeline-model-fe'
+dbutils.jobs.taskValues.set(key="new_model_uri", value=model_uri)
```

Ruff (0.7.0)

Undefined name `dbutils`

(F821)